config/config-example.js

```diff
@@ -288,12 +288,12 @@ exports.restrictLinks = false;
 exports.chatmodchat = false;
 /**
  * battle modchat - default minimum group for speaking in battles; changeable with /modchat
```

Comment:
In hindsight it would make more sense to call these `modbattle` and `modpm`, but I guess the ship has sailed. Or maybe we could support them as the new names for these?